examples/make_eeg_bids.py

```diff
@@ -0,0 +1,173 @@
+"""
+=========================
+Use MNE-Bids for EEG Data
+=========================
+
+In this example, we use MNE-Bids to create a BIDS-compatible directory of EEG
+data. Specifically, we will follow these steps:
+
+1. Download some EEG data from the Open Science Framework (https://osf.io)
+2. Load the data, extract information, and save in a new BIDS directory
+3. Check the result and compare it with the standard
+
+"""
+
+# Authors: Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+# License: BSD (3-clause)
+
+###############################################################################
+# We are importing everything we need for this example:
+import os
+
+from mne import find_events
+from mne.io import read_raw_brainvision
+
+from mne_bids import raw_to_bids
+from mne_bids.datasets import download_matchingpennies_subj
```

Comment:
this is not yet there.

Also, if I remember correctly, this dataset was already in BIDS format. How do you propose to use it in the example in that case?